src/test/kotlin/bsu/cc/constraints/ConstraintsTest.kt

```diff
@@ -0,0 +1,105 @@
+package bsu.cc.constraints
```

Comment:
These tests indeed are quite nice! it's very readable and has way less boilerplate than a lot of testing frameworks.

Reply:
you should see clojure tests ;)